money/contrib/django/models/fields.py

```diff
@@ -13,11 +13,17 @@ def currency_field_name(name):
     return "%s_currency" % name
 
 
+def currency_field_db_column(db_column):
```

Comment:
why did you not use `currency_field_name`? I'm pretty sure that is what is the "db column name"